packages/components/src/components/tick-picker/tick-picker.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import Button              from '../button';
+import Icon                from '../icon';
+
+const TickPicker = ({ min_value, max_value, onSubmit, label }) => {
+    const normalizedTick = (tick) => ((parseInt(tick) < 10 ? '0' : '') + tick);
+    const [tick_value, setTickValue] = useState(parseInt(min_value));
+
+    const handleDecrease = () => {
+        if (tick_value - 1 >= min_value) {
+            setTickValue(tick_value - 1);
+        }
+        
+    };
+    const handleIncrease = () => {
+        if (tick_value + 1 <= max_value) {
+            setTickValue(tick_value + 1);
+        }
+        
+    };
+    const handleClick = () => {
+        onSubmit({
+            target: {
+                value: tick_value,
+                name : 'submit',
+            },
+        });
+    };
+    return (
+        <div className='tick-picker'>
+            <div className='tick-picker--calculation'>
+                <Button curved className='operator' onClick={handleDecrease}>
+                    <Icon icon='IcMinus' custom_color='var(--text-prominent)' />
+                </Button>
+                <div className='tick-holder'>
+                    <span className='tick-holder--counter'>{normalizedTick(tick_value)}</span>
+                    <span className='tick-holder--text'>Ticks</span>
```

Comment:
`Ticks` have to be dynamic, with translations in mind, please add the prop and make it dynamic.

Reply:
Also bear in mind, that the `string` has to be `Tick` when `1` is the selected tick as its considered a singular noun.

Reply:
what approach should I take? should I get the dynamic content from the props and remove the letter `s` from the end or anything else?

Reply:
@arash-fs , I've discussed with @oskar-binary  and he recommends that we pass the `string` as a `prop`. 

That way we can keep the component more general instead just for `Ticks`. It could potentially be used as a increment/decrement number picker since we're also adding props for `min` and `max`. 

So the `string` below the number picker can be any text we want it to be instead of just `Tick`.

Maybe u can pass two additional `string` props, for `singular` (e.g - Tick) and `plural` (e.g - Ticks). 

And maybe we can rename the component into NumberPicker instead of TickPicker?

Reply:
per my conversation with @reza-fs I going to add two props for singular and multiple based content. for example:
`singular_label='tick' plural='ticks' `